packages/react-native-payments/lib/js/helpers/index.js

```diff
@@ -176,13 +176,14 @@ export function validatePaymentMethods(methodData): Array {
   return serializedMethodData;
 }
 
+
 export function validateDisplayItems(displayItems, errorType = Error): void {
   // Check that the value of each display item is a valid decimal monetary value
   if (displayItems) {
     displayItems.forEach((item: PaymentItem) => {
       const amountValue = item && item.amount && item.amount.value;
 
-      if (!amountValue) {
+      if (!amountValue && amountValue !== 0) {
```

Comment:
I would rather skip this check here and 
1. add it to `isValidDecimalMonetaryValue` check
or 
2. create `isDefined` constraint function to be explicit about constraint here.